lib/cache/PackFileCacheStrategy.js

```diff
@@ -639,10 +639,15 @@ class PackContentItems {
 				} catch (e) {
 					rollback(s);
 					if (e === NOT_SERIALIZABLE) continue;
-					logger.warn(
-						`Skipped not serializable cache item '${key}': ${e.message}`
+					const message = `Skipped not serializable cache item: ${e.message}`;
+					if (e.message.includes("ModuleBuildError")) {
+						logger.log(message);
+					} else {
+						logger.warn(message);
```

Comment:
maybe use another text, it could be weird for user from output why `Skipped not serializable cache item` have different log level.